src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -186,12 +186,13 @@ module.exports.commands = [{
             // validate that we have changed the url
             browser.url(function (responseInner) {
                 sanityCheck(self, responseInner);
+                // due to JENKINS-37925 the following is de-activated
                 // is the "full log" link gone?
-                self.fullLogButtonNotPresent();
+                // self.fullLogButtonNotPresent();
                 // is the progressbar visible and not the code?
-                self.validateLoading();
+                // self.validateLoading();
                 // did we changed the url on  change?
-                self.assert.equal(responseInner.value.includes('start=0'), true);
+                // self.assert.equal(responseInner.value.includes('start=0'), true);
```

Comment:
Can we delete this stuff if it's no longer being used? Otherwise the whole thing just ends up being a noisy mess.